### PR TITLE
[Request]onElementChange

### PR DESCRIPTION
**Pull request by FileEX**
This event instead onElementModelChange, onElementDimensionChange, onElementInteriorChange etc.

_**Parameters**_

string whatChange, int oldValue, int newValue

whatChange:
- Model
- Interior
- Dimension

### CODE REVIEW

File: Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp [hunk 1]
```diff
@@ -10907,6 +10933,21 @@ bool CStaticFunctionDefinitions::GetAccountsBySerial(const SString& strSerial, s
     return true;
 }
 
+bool CStaticFunctionDefinitions::GetAccountID(CAccount* pAccount, int& ID)
```
Comment (CrosRoad95) [inline Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: wut ?

File: Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp [hunk 2]
```diff
@@ -1734,10 +1752,11 @@ bool CStaticFunctionDefinitions::SetElementModel(CElement* pElement, unsigned sh
             if (!CObjectManager::IsValidModel(usModel))
                 return false;
             CLuaArguments Arguments;
+            Arguments.PushString("Model"); // what change? -- Model
             Arguments.PushNumber(pObject->GetModel());            // Get the old model
             pObject->SetModel(usModel);                           // Set the new model
             Arguments.PushNumber(usModel);                        // Get the new model
-            pObject->CallEvent("onElementModelChange", Arguments);
```
Comment (CrosRoad95) [inline Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: this will break current scripts

**Comment by CrosRoad95** [comment]
upstream your respository.

**Review by CrosRoad95 — CHANGES_REQUESTED** [review]
Upstream your branch.

File: Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp (hunk 1 above)
Comment (FileEX) [reply Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: Idk, i don't added this.

**Pull request closed without merging**